cgroups/fs/memory.go

```diff
@@ -26,7 +26,7 @@ func (s *MemoryGroup) Set(d *data) error {
 	}()
 
 	// Only set values if some config was specified.
-	if d.c.Memory != 0 || d.c.MemoryReservation != 0 || d.c.MemorySwap != 0 {
+	if d.c.Memory != 0 || d.c.MemoryReservation != 0 || d.c.MemorySwap != 0 || d.c.OomKillDisable {
```

Comment:
Break out OomKillDisable into its own if statement. It shouldn't be bundled with these here (neither should Reservation and Memory for that matter, but that's out of scope :) ).